internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -157,6 +158,23 @@ func convertReplicationSpecs(ctx context.Context, input *[]admin.ReplicationSpec
 	return &tfModels
 }
 
+func selectContainerIDs(spec *admin.ReplicationSpec20240805, allIDs map[string]string) map[string]string {
+	containerIDs := map[string]string{}
+	regions := spec.GetRegionConfigs()
+	for i := range regions {
+		regionConfig := regions[i]
+		providerName := regionConfig.GetProviderName()
+		key := containerIDKey(providerName, regionConfig.GetRegionName())
+		value := allIDs[key]
+		// Should be no hard failure if not found, as it is not required for TENANT
```

Comment:
resolveContainerIDs will raise an error and always called before this function.
Do you think it is still worth it to do that check here? It will be kind of double logic

Reply:
thanks for the context. then it's ok to leave it as it is, thanks